server/src/com/cloud/server/ManagementServerImpl.java

```diff
@@ -696,6 +696,8 @@
 
     static final ConfigKey<Integer> vmPasswordLength = new ConfigKey<Integer>("Advanced", Integer.class, "vm.password.length", "6",
                                                                                       "Specifies the length of a randomly generated password", false);
+    static final ConfigKey<Integer> sshKeyLength = new ConfigKey<Integer>("Advanced", Integer.class, "ssh.key.length",
+            "2048", "User SSH key length (bit)", true, ConfigKey.Scope.Account);
```

Comment:
Might the description be more meaningful for average user, like "Specifies minimal? amount of SSH key length permitted". Could you please take a look at this. May be it can be improved.